internal/rbd/manager.go

```diff
@@ -334,6 +334,71 @@ func (mgr *rbdManager) GetVolumeGroupSnapshotByID(
 	return vgs, nil
 }
 
+func (mgr *rbdManager) GetVolumeGroupSnapshotByName(
+	ctx context.Context,
+	name string,
+) (types.VolumeGroupSnapshot, error) {
+	pool, ok := mgr.parameters["pool"]
+	if !ok || pool == "" {
+		return nil, errors.New("required 'pool' option missing in volume group parameters")
+	}
+
+	// volumeNamePrefix is an optional parameter, can be an empty string
+	prefix := mgr.parameters["volumeNamePrefix"]
+
+	// journalPool is an optional parameter, use pool if it is not set
+	journalPool, ok := mgr.parameters["journalPool"]
+	if !ok || journalPool == "" {
+		journalPool = pool
+	}
```

Comment:
as i see we are not using journalPool for rados, should we drop it?

Reply:
yeah, let's do that